prepare_GBM_data.R

```diff
@@ -0,0 +1,195 @@
+# Script prepares GBM expression data for use in pipelines
+# For GBM array data: convert sample names and remove duplicate individuals
+# For GBM seq data: filter all seq data for GBM samples only, convert genes IDs
+
+# Steven Foltz July 2021
+
+option_list <- list(
+  optparse::make_option("--seq_input",
+                        default = NULL,
+                        help = "TCGA sequencing expression input file path"),
+  optparse::make_option("--array_input",
+                        default = NULL,
+                        help = "refine.bio microarray expression input file path"),
+  optparse::make_option("--metadata_input",
+                        default = NULL,
+                        help = "refine.bio aggregated metadata JSON file path"),
+  optparse::make_option("--array_output",
+                        default = NULL,
+                        help = "Processed microarray data output file path"),
+  optparse::make_option("--seq_output",
+                        default = NULL,
+                        help = "Processed sequencing data output file path"),
+  optparse::make_option("--clinical_input",
+                        default = NULL,
+                        help = "Clinical information input file path (Excel file)"),
+  optparse::make_option("--clinical_output",
+                        default = NULL,
+                        help = "Clinical information output file path (.tsv)"),
+  optparse::make_option("--overwrite",
+                        action = "store_true",
+                        default = FALSE,
+                        help = "Overwrite existing output files [default: %default]")
+)
```

Comment:
If the rest of this script is _very much tied_ to the GBM data and the cleaning steps are not generalizable, what do you think the advantages of using `optparse` are here?

Reply:
Sooo right that this is GBM specific. The main benefit I see is being able to use `check_options()` from `util/option_functions.R`.